src/events.rs

```diff
@@ -312,3 +312,11 @@ pub enum VirtualKeyCode {
     WebStop,
     Yen,
 }
+
+#[derive(Default, Debug, Clone, Copy)]
+pub struct Mods {
+  pub shift: bool,
+  pub ctrl: bool,
+  pub alt: bool,
+  pub supr: bool
```

Comment:
I looked into this and super key seems to be the most common platform neutral name for the windows / command key. Super is reserved so I named it supr.